Tests/Microsoft.AppCenter.Test.Windows/AppCenterTest.cs

```diff
@@ -644,7 +644,16 @@ public void ParsingFailIfWhiteSpaceBeforeFirstSecret()
             var appSecret = Guid.NewGuid().ToString();
             var platformId = "uwp";
             var secrets = $" {platformId}={appSecret};ios=anotherstring";
-            var parsedSecret = AppCenter.GetSecretAndTargetForPlatform(secrets, platformId);
+            var parsedSecret = "";
+            try
+            {
+                parsedSecret = AppCenter.GetSecretAndTargetForPlatform(secrets, platformId);
+                Assert.Fail("It should be failed.");
+            } 
+            catch (Exception e)
+            {
+                Assert.AreEqual(e.GetType(), typeof(AppCenterException));
+            }
             Assert.AreNotEqual(appSecret, parsedSecret);
```

Comment:
We don't need this line then, and we don't need to keep parsedSecret as a variable.